perun/profile/factory.py

```diff
@@ -203,7 +211,26 @@ def all_resources(self):
                 snapshot_number = collectable_properties.get('snapshot', 0)
                 yield snapshot_number, collectable_properties
 
-    def all_models(self):
+    def get_models(self, models_strategy):
```

Comment:
Fix: The name is too similar to `all_models`, I would be confused which one to use. Rename this e.g. to `all_filtered_models` or `filtered_all_models`.